src/features/applications/components/ApplicationReviewActions/index.tsx

```diff
@@ -0,0 +1,42 @@
+"use client";
+
+import { useApplicationReview } from "../../hooks/useApplicationReview";
+import { ApplicationStatusBadge } from "./ApplicationStatusBadge";
+import { ReviewActionButton } from "./ReviewActionButton";
+
+type Props = {
+  projectId: string;
+};
+
+export function ApplicationReviewActions({ projectId }: Props) {
+  const {
+    status,
+    isAdmin,
+    userCanRevoke,
+    onApprove,
+    onRevoke,
+    revokeIsPending,
+    approveIsPending,
+  } = useApplicationReview({ projectId });
+
+  if (!status) return null;
+
+  if (!isAdmin) return <ApplicationStatusBadge status={status} />;
```

Comment:
Remove this line and pass `isAdmin` to `ReviewActionButton`.

Reply:
done